.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Guide (you can delete this section)
+* Ensure your PR title and branch name starts with an issue number and a dash. E.g. `583 - Add tapestry zoom feature`
+* Ensure your PR title has "Bug fix:" after the issue number and dash if it's a bug fix. E.g. `583 - Bug fix: Fix tapestry not zooming`
+* Ensure you fill in the sections below, if applicable. Do NOT delete the sections below.
+* Delete this section after completing the above steps and before creating the PR
+## Changes
+* Write down exactly what changes this PR makes to the base branch. This will be used as the commit message once your PR is merged in. Note that all of your commit messages will be deleted when merging this PR. Please delete this line
+## Issue Linkage
+* Include a [closing keyword](https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords) here so that the issue automatically closes when this PR gets merged in. E.g. `Fixes #582`. Please delete this line
+## PR Dependency
+* If your PR is dependent on another PR, put that PR # in the description. E.g. `Depends on: #708`. If no dependency, just enter `N/A`. Please delete this line.
```

Comment:
Possible suggestion. Instead of all the text under Issue Linkage and PR Dependency. Just have 
Fixes # (issue #)
Depends on: (PR # or N/A)